portal/integration/blog.spec.js

```diff
@@ -0,0 +1,21 @@
+context("Blog", () => {
+    beforeEach(() => {
+        cy.visit("/");
+    });
+
+    it("Blog should work", () => {
+        cy.getByTestid("header-Blog").click();
```

Comment:
```suggestion
        cy.getByTestid("header-Blogg").click();
```
Feiler nå som tittelen er endret